Enterprise-CMCS/cmcs-eregulations
# EREGCSC-2320 -- Enable active state for selected filter item

Resolves [EREGCSC-2320](https://jiraent.cms.gov/browse/EREGCSC-2320)

**Description**

The Subject Selector in the left sidebar of the Policy Repository page already has bolded styles added to the subject item when you hover over a subject with the cursor, when you focus the subject item using keyboard navigation, and briefly after you select the subject item.  Once you click away from the subject item, however, the bolded style is removed.

This PR will make that bolded style persist as long as that subject is selected.

**This pull request changes:**

- Adds `--selected` styles to `sidebar-li__button`
- watches `$route.query.subjects` to determine is a selected subject needs the `--selected` style

**Steps to manually verify this change:**

1. Visit Policy Repository at the experimental deployment
2. Add some subjects using the subject selector in the right sidebar
3. Ensure that the bolded style remains even after you click away after selecting the subject
4. Ensure that the bolded style does not interfere with the subject typeahead styles, and is restored once the typeahead input is cleared
5. Ensure that the bolded style is applied to selected subjects when visiting a URL with several subjects already selected
